app/components/blacklight/start_over_button_component.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+module Blacklight
```

Comment:
Please don't use the namespace already in use in Blacklight (`Blacklight::StartOverButtonComponent`) . This is confusing. Instead, define `StartOverButtonComponent` at the root level (and extend `Blacklight::StartOverButtonComponent`)

Reply:
If you don't use the blacklight namespace you can't override the StartOverButton in Blacklight.

Reply:
I don't think you need to override it.  Make your own class.  call the new class in `app/views/catalog/_start_over.html.erb` This makes it very explicit about what you are doing.

Reply:
Then we have to also override the constraints component as well. https://github.com/projectblacklight/blacklight/blob/d566efebb088ef87bffeb5a6b5a12c5017adda5f/app/components/blacklight/constraints_component.rb#L28

Reply:
@dnoneill you can pass that class as an argument to ContstraintsComponent.  No overriding is required.

Reply:
Thanks for the tip. Updated.